Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -1050,10 +1083,20 @@ def captureImageFromView(self, view, filename, transparentBackground=False):
       imageClipper.Update()
       capturedImage = imageClipper.GetOutput()
 
-    writer = self.createImageWriter(filename)
-    writer.SetInputData(self.addWatermark(capturedImage))
-    writer.SetFileName(filename)
-    writer.Write()
+    capturedImage = self.addWatermark(capturedImage)
+    if isinstance(volumeNode, slicer.vtkMRMLVectorVolumeNode):
+      ijkToRas = vtk.vtkMatrix4x4()
+      ijkToRas.SetElement(0, 0, -1)
+      ijkToRas.SetElement(1, 1, -1)
+      volumeNode.SetIJKToRASMatrix(ijkToRas)
+      volumeNode.SetAndObserveImageData(capturedImage)
+    else:
+      raise ValueError("Invalid vector volume node.")
```

Comment:
First check if volumeNode is not None, then check the volume type and raise an exception if it is not correct.